src/serac/physics/thermal_solid.hpp

```diff
@@ -63,21 +63,21 @@ class ThermalSolid : public BasePhysics {
    *
    * @param[in] kappa The thermal conductivity
    */
-  void SetConductivity(std::unique_ptr<mfem::Coefficient>&& kappa) { therm_solver_.setConductivity(std::move(kappa)); };
+  void setConductivity(std::unique_ptr<mfem::Coefficient>&& kappa) { therm_solver_.setConductivity(std::move(kappa)); };
 
   /**
    * @brief Set the density
    *
    * @param[in] rho The density coefficient
    */
-  void SetDensity(std::unique_ptr<mfem::Coefficient>&& rho) { therm_solver_.setDensity(std::move(rho)); };
+  void setDensity(std::unique_ptr<mfem::Coefficient>&& rho) { therm_solver_.setDensity(std::move(rho)); };
```

Comment:
Would you consider "mass density" instead of "density"? I could see nomenclature collisions with topology optimization (which to be extra confusing also uses rho), although within LiDO I'm trying to either call it design density or local volume fraction.

Reply:
Sounds reasonable to me!